src/operator.py

```diff
@@ -0,0 +1,368 @@
+from drone import Drone
+from sensor import Sensor
+import roslibpy
+import argparse
+#roslaunch rosbridge_server rosbridge_websocket.launch
+
+#####################
+# Global Parameters #
+#####################
+# Allows option to type --ip and specify ip of the server through the command line. Ex: python3 operator.py --ip 0.0.0.0
+parser = argparse.ArgumentParser(description='Starts the operator of the server.')
+parser.add_argument('--ip', type=str, default='54.161.15.175')
+args = parser.parse_args()
+
+# HOST ip parameter
+HOST = args.ip
+
+####################
+# Global Variables #
+####################
+
+drones = dict() # Global map between drone IDs and drone instances
+sensors = dict() # Global map between sensor IDs and sensor instances
+drone_names = dict() # Global map between drone names and drone IDs
+sensor_names = dict() # Global map between sensor names and sensor IDs
+all_topics = dict() # Global map of topic names to topic types
+# If an id of 0 is passed in, it refers to all drones, sensors, and global topics
+next_id = 1 # ID to assign next drone or sensor
+
+###################################
+# Set up and boot Roslibpy server #
+###################################
+
+ROS_master_connection = roslibpy.Ros(host=HOST, port=9090)
+services = []
```

Comment:
Add ROS_master_connection and services to the global variables? Or at least services?

Reply:
Move services.  I think master connection belongs under booting the server